android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1532,10 +1535,27 @@ public static SpannableString createSpannableString(
             String text, ClickableSpan clickListener, int startIndex, int endIndex, int flags) {
         assert null != text;
         SpannableString spannableString = new SpannableString(text);
-        spannableString.setSpan(clickListener, startIndex, endIndex, flags);
+        if (startIndex >= 0 && endIndex > startIndex && endIndex < text.length())
+            spannableString.setSpan(clickListener, startIndex, endIndex, flags);
         return spannableString;
     }
 
+    public static Spannable createSpannableUsingHTML(
+            Context context, @StringRes int resId, View.OnClickListener onClickListener) {
+        String htmlString =
+                String.format(context.getResources().getString(resId), "<a href=\"\">", "<\\a>");
```

Comment:
why `<//a>`?

Reply:
a closing tag for <a>, single slash cannot be used inside the string literal

Reply:
Got it, this can be moved to xml with no translation.